src/go/guestagent/pkg/tracker/apitracker.go

```diff
@@ -88,6 +88,8 @@ func (a *APITracker) Add(containerID string, portMap nat.PortMap) error {
 	for portProto, portBindings := range portMap {
 		var tmpPortBinding []nat.PortBinding
 
+		log.Infof("PortProto: %+v, PortBindings: %+v\n", portProto, portBindings)
```

Comment:
Is `Infof` infrequent enough when everything is going right and the user keeps this running over a week?

Reply:
Oops, that should be `Debugf`